_posts/dev/2014-08-02-kura-dio-apis.md

```diff
@@ -128,6 +130,205 @@ button.setInputListener(new PinListener(){
 		}
 });
 
+```
+### Accessing a GPIO Pin with GPIOService for a switch a Led. 
+
+Inside the **jdk.dio.properties** file:
+
+```
+...
+6 = deviceType: gpio.GPIOPin, pinNumber:6, name:GPIO06
+...
+```
+
+Then in the code:
+
+```java
+
+private GPIOService myservice;
+
+KuraGPIOPin pin = this.myservice.getPinByTerminal(6);
+
+// to allow pin configuration use
+//pin = this.myservice.getPinByTerminal(this.options.isConfigPin());
```

Comment:
Since in the meta type you are allowing the user to specify the GPIO pin to use, I believe  the code should follow the same pattern.

Reply:
This comment needs to be addressed